drupal/sites/all/themes/antistatique/assets/sass/atoms/_button.scss

```diff
@@ -190,13 +200,13 @@
 }
 
 .btn-xxs {
-  font-size: .8em;
+  font-size: 0.8em;
   line-height: 1;
 
   @media only screen and (min-width: $screen-sm-min) {
-    font-size: .6em;
-    line-height: 1;
     padding: 5px 6px 4px;
```

Comment:
This is some old code, I won't change it back. Hoping on a redesign to happen shortly 😄